src/Form/AdminSettingsForm.php

```diff
@@ -102,50 +95,37 @@ private function buildCredentialForm() {
     $form['account_name'] = [
       '#type' => 'textfield',
       '#title' => t('Account Name'),
+      '#description' => t('Your Lift subscription\'s account name.'),
       '#default_value' => $credential_settings['account_name'],
       '#required' => TRUE,
     ];
     $form['customer_site'] = [
       '#type' => 'textfield',
       '#title' => t('Customer Site'),
+      '#description' => t('Current site\'s site name. WARNING: different sites must use different value here, even between a staging and a production of the same site.'),
       '#default_value' => $credential_settings['customer_site'],
-    ];
-    $form['api_url'] = [
-      '#type' => 'textfield',
-      '#title' => t('API URL'),
-      '#default_value' => $credential_settings['api_url'],
-      '#required' => TRUE,
-    ];
-    $form['access_key'] = [
-      '#type' => 'textfield',
-      '#title' => t('API Access Key'),
-      '#default_value' => $credential_settings['access_key'],
-      '#required' => TRUE,
-    ];
-    $form['secret_key'] = [
-      '#type' => 'password',
-      '#title' => t('API Secret Key'),
-      '#default_value' => $credential_settings['secret_key'],
-      '#required' => empty($credential_settings['secret_key']),
-      '#description' => !empty($credential_settings['secret_key']) ? t('Only necessary if updating') : '',
-    ];
-    $form['js_path'] = [
-      '#type' => 'textfield',
-      '#title' => t('JavaScript Path'),
-      '#default_value' => $credential_settings['js_path'],
       '#required' => TRUE,
     ];
     $form['assets_url'] = [
       '#type' => 'textfield',
       '#title' => t('Assets URL'),
-      '#default_value' => $credential_settings['assets_url'],
+      '#field_prefix' => 'https://',
```

Comment:
Yes it trims and converts, cowboy style. =P

I would consider those mechanisms just precautions, and wouldn't expect them to catch a lot mistakes. This is because in the current process, we [generate](https://backlog.acquia.com/browse/CSO-3578?focusedCommentId=637294&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-637294) and customers receive all credential info. I wouldn't really worry about a customer going off-course from simple instructions.